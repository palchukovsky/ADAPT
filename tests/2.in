SCOPE x {
	DECLARE y;
	ACCESS y;
}