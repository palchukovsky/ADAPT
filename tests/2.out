LINE 3 ACCESS ::x::y
